docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -580,7 +580,7 @@ To better understand the snap connections, read the [interface management](https
     ```
 
 ### EdgeX UI
-| [Installation][edgex-ui] | [Managing Services] | [Debugging] | [Source](https://github.com/edgexfoundry/edgex-ui-go/tree/{{edgexversion}}/snap) |
+| [Installation][edgex-ui] | [Managing Services] | [Debugging] | |
```

Comment:
do these tables get rendered with an extra column? the `|  |` from removing the snap?

Reply:
An empty column for now. Doesn't matter as the docs will be removed by Canonical soon.